rlp/types.go

```diff
@@ -121,6 +121,20 @@ func (i Item) IP() (net.IP, error) {
 	}
 }
 
+func (i Item) Bytes32() ([32]byte, error) {
+	if len(i.d) != 32 {
+		return [32]byte{}, errors.New("must be exactly 32 bytes")
+	}
+	return *(*[32]byte)(i.d), nil
+}
+
+func (i Item) Bytes65() ([65]byte, error) {
```

Comment:
What's special about a 65-byte array? If this is only meant for the signature (at least within this PR it's the only reference), then perhaps call it `Signature` or something?

Reply:
I was considering that, but I think I like the explicitness of a method name that matches the return type.